roles/confluent.variables/defaults/main.yml

```diff
@@ -58,6 +58,17 @@ common_role_completed: false
 
 proxy_env: {}
 
+# Archive file options that don'e use apt/yum
```

Comment:
please add annotation in the form described in a previous comment!

Reply:
Yep, fixed in the latest commit b7c40508c723c2e8fced5a6ba75f2634eb0d6f15